score-vmware/src/main/java/io/cloudslang/content/vmware/services/helpers/ResponseHelper.java

```diff
@@ -0,0 +1,52 @@
+package io.cloudslang.content.vmware.services.helpers;
+
+import com.vmware.vim25.*;
+import io.cloudslang.content.vmware.connection.ConnectionResources;
+import io.cloudslang.content.vmware.connection.helpers.WaitForValues;
+import io.cloudslang.content.vmware.constants.Outputs;
+import io.cloudslang.content.vmware.entities.VmParameter;
+import io.cloudslang.content.vmware.utils.ResponseUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Mihai Tusa.
+ * 3/22/2016.
+ */
+public class ResponseHelper {
+    public Map<String, String> getResultsMap(ConnectionResources connectionResources, ManagedObjectReference task,
+                                             String successMessage, String failureMessage)
```

Comment:
pass ConnectionResources and ManagedObjectReference objects in a constructor for ResponseHelper